src/registries/data_validation_registry.ts

```diff
@@ -706,6 +736,16 @@ function checkValueIsNumber(value: string): boolean {
   return valueAsNumber !== undefined;
 }
 
+function isValidFormula(value: string): boolean {
+  if (!value.startsWith("=")) {
+    return true;
+  }
```

Comment:
I'd argue that a string not beginning by `=` is not a valid formula, sicne it's not a formula at all ...